app_test.go

```diff
@@ -334,6 +342,28 @@ func TestBashCompletionOptions(t *testing.T) {
 			Args:            "--completion-bash two --flag-3 opt4 --",
 			ExpectedOptions: []string{"--help", "--flag-2", "--flag-3", "--flag-0", "--flag-1"},
 		},
+
+		// Args complete
+		{
+			// After a command with arg options
+			Args:            "--completion-bash three ",
+			ExpectedOptions: []string{"arg-2-opt-1", "arg-2-opt-2", "arg-3-opt-1", "arg-3-opt-2"},
```

Comment:
IIUC, these hints shouldn't be available here, as (unlike flags) args are positional. At this point it should be completing for `arg-1` *only* which has no hints and thus should not be completing.